src/App/index.js

```diff
@@ -0,0 +1,22 @@
+import React from "react"
+import { Link } from "gatsby"
+import styled from "styled-components"
+
+import Layout from "../components/layout"
+// import Image from "../components/image"
+// import SEO from "../components/seo"
+
+const IndexPage = props => {
```

Comment:
I did this to simplify some of the errors that were happening because it was trying to render `SEO`